src/components/07-form/templates/address-form.njk

```diff
@@ -17,7 +18,7 @@
         <div class="usa-input-grid usa-input-grid-small">
           <label for="state">State</label>
           <select id="state" name="state">
-            <option value></option>
+            <option value>- Select -</option>
```

Comment:
I realized, I set it to `value=""` in the other file, but this already existed as `value`. So I think this one is correct and I have to change the other one.